odoo/sql_db.py

```diff
@@ -223,7 +223,7 @@ def execute(self, query, params=None, log_exceptions=None):
             res = self._obj.execute(query, params)
         except Exception as e:
             if self._default_log_exceptions if log_exceptions is None else log_exceptions:
-                _logger.error("bad query: %s\nERROR: %s", self._obj.query or query, e)
+                _logger.error("bad query: %s\nERROR: %s", self._obj.query.decode() or query, e)
```

Comment:
@rco-odoo what about this one? The query used to be `bad query: b'INSERT ...'` with `\n` escaped

Reply:
removed as not really linked to this PR